tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/SwiftPrinter.swift

```diff
@@ -89,11 +129,59 @@ public class SwiftPrinter: BasePrinter {
         writeLine("}")
     }
 
+    private func printCodableMethods(for properties: [SwiftStruct.Property], additionalProperties: SwiftStruct.Property) throws {
```

Comment:
Please see the associated tests for a preview of the output:
https://github.com/DataDog/dd-sdk-ios/pull/399/files#diff-fcc08ef045de4c9d692921649cfe668e0900e3900473a1e5702383cc67c37efa